frontend/src/container/QueryBuilder/tests/FilterLabel.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import i18n from 'ReactI18';
+import store from 'store';
+
+import { FilterLabel } from '../components';
+
+describe('QueryBuilder', () => {
```

Comment:
can you please write filter specific name `QueryBuilder` seems like a generic one